riotfile.py

```diff
@@ -297,5 +297,25 @@ def select_pys(min_version=min(SUPPORTED_PYTHON_VERSIONS), max_version=max(SUPPO
             pys=select_pys(min_version=3.6),
             command="pytest tests/contrib/asgi",
         ),
+        Venv(
+            name="fastapi",
+            command="pytest {cmdargs} tests/contrib/fastapi",
+            venvs=[
+                Venv(
+                    pys=select_pys(min_version=3.6),
+                    pkgs={
+                        "fastapi": [latest],
```

Comment:
fastapi has a ton of releases in the past few years it'd be good to include a few more here. Typically we do every other version if there are a lot but there have been like 60 releases in the past few years 😆. Perhaps we can do every 4 releases until some cut-off?